src/crimson/osd/pg_backend.h

```diff
@@ -79,10 +80,10 @@ class PGBackend
     std::set<pg_shard_t> pg_shards,
     crimson::osd::ObjectContextRef &&obc,
     ceph::os::Transaction&& txn,
-    const MOSDOp& m,
+    OSDOpParams&& osd_op_p,
     epoch_t min_epoch,
     epoch_t map_epoch,
-    eversion_t ver);
+    vector<pg_log_entry_t> logv);
```

Comment:
why do you prefer passing it by value over by reference (or better off rvalue reference)?